recipes/nvmolkit/variants.yaml

```diff
@@ -0,0 +1,3 @@
+librdkit-dev:
```

Comment:
Noticed the feedstock [conversion job failed]( https://github.com/conda-forge/admin-requests/actions/runs/18794165898/job/53630841555#step:7:1795 )

The issue is `-` isn't valid in variant names. So we would want to change `-` here to `_`

Have done this in PR: https://github.com/conda-forge/staged-recipes/pull/31276

That should hopefully fix the issue